tools/dgeni/common/dgeni-accessors-parse.ts

```diff
@@ -0,0 +1,54 @@
+import {getTypeText} from 'dgeni-packages/typescript/services/TsParser/getTypeText';
+import {getDecorators} from 'dgeni-packages/typescript/services/TsParser/getDecorators';
+import {
+  ClassDeclaration,
+  SyntaxKind,
+  GetAccessorDeclaration,
+  SetAccessorDeclaration,
+  Symbol,
+  displayPartsToString
+} from 'typescript';
+import {CategorizedPropertyMemberDoc} from '../processors/categorizer';
+
+/**
+ * Accessors are not parsed properly within the latest dgeni-packages package.
+ * This function re-analyzes the accessors and resolves the necessary API docs information.
+ */
+export function dgeniAccessorsParse(propertyDoc: CategorizedPropertyMemberDoc) {
+  if ((propertyDoc.isGetAccessor || propertyDoc.isSetAccessor) && !propertyDoc.type) {
+    const classDeclaration = propertyDoc.containerDoc.symbol.valueDeclaration as ClassDeclaration;
+
+    // Walk through every TypeScript member node of the class declaration and filter for accessors
+    // with a name that match the current property member. For accessors there may be two
+    // declarations with the same name but with different type information.
+    classDeclaration.members
+      .filter((member: any) => member.kind === SyntaxKind.GetAccessor ||
+        member.kind === SyntaxKind.SetAccessor)
+      .filter((member: any) => member.name && (member.name as any).text === propertyDoc.name)
+      .forEach((member: GetAccessorDeclaration|SetAccessorDeclaration) => {
+        if (member.type) {
+          propertyDoc.type = getTypeText(member.type, []);
+          propertyDoc.decorators = getDecorators(member);
+        } else if (member.parameters.length === 1 && member.parameters[0].type) {
+          const type = member.parameters[0].type;
+          if (type) {
+            propertyDoc.type = getTypeText(type, []);
+            propertyDoc.decorators = getDecorators(member);
+          }
+        }
+
+        // If the current property document doesn't have any description set and the current
+        // TypeScript has a symbol from the TypeChecker, we can try to find a documentation
+        // comment on that symbol.
+        if (!propertyDoc.description && (member as any).symbol) {
```

Comment:
If you need the `as any` here, is `GetAccessorDeclaration|SetAccessorDeclaration` the right type for member?

Reply:
Yeah this is the correct type. It just looks like TypeScript isn't exposing the `symbol` property in their typings. The same is happening on Dgeni as well ([see here](https://github.com/angular/dgeni-packages/blob/master/typescript/src/services/TsParser/index.ts#L56))